FizzBuzz.cs

```diff
@@ -1,24 +1,23 @@
 public class Solution
 {
-    private bool Isdivisibleby (int num1,int div)
-    {  
-        return num1 % div == 0;
+    private bool IsDivisibleBy (int dividend, int divisor)
+    {
+        return dividend % divisor == 0;
     }
     public IList<string> FizzBuzz(int n)
     {
         List<string> lista = new List<string>();
-
-        for (int i = 1; i <= n; i++)
-        {
-            string result = i switch
+        for( int i = 1; i <= n; i++)
         {
-            _ when Isdivisibleby(i, 3) && Isdivisibleby(i, 5) => "FizzBuzz",
-            _ when Isdivisibleby(i, 3) => "Fizz",
-            _ when Isdivisibleby(i, 5) => "Buzz",
-            _ => i.ToString()
+            string result= i switch
+            {
+                _ when IsDivisibleBy(dividend: i, divisor: 3) && IsDivisibleBy(dividend: i, divisor: 5) => "FizzBuzz",
+                _ when IsDivisibleBy(dividend: i, divisor: 3) => "Fizz",
+                _ when IsDivisibleBy(dividend: i, divisor: 5)=> "Buzz",
```

Comment:
Nice named arguments! :) I love using those, I always try to encourage people to use them in the PR I review.

Line 16 has the tiniest formatting issue. 
```suggestion
                _ when IsDivisibleBy(dividend: i, divisor: 3) && IsDivisibleBy(dividend: i, divisor: 5) => "FizzBuzz",
                _ when IsDivisibleBy(dividend: i, divisor: 3) => "Fizz",
                _ when IsDivisibleBy(dividend: i, divisor: 5) => "Buzz",
```